.github/workflows/rust.yml

```diff
@@ -232,9 +232,21 @@ jobs:
     name: "Run sqllogictest with Postgres runner"
     needs: [linux-build-lib]
     runs-on: ubuntu-latest
+    services:
+      postgres:
```

Comment:
this starts the container using the code from `integration-tests` (above in this file) rather than running docker from within the harness